src/Api/VendorCredits.php

```diff
@@ -13,6 +13,12 @@ public function applyToBill($vendorCreditId, $data)
         return true;
     }
 
+    public function getList(array $filters = [])
```

Comment:
It method not needed here, because of bug in documentation: https://www.zoho.com/books/api/v3/vendorcredits/#list-vendor-credits
In DOC example response with key="vendorcredits", but in real API response key is "vendor_credits" (usual API_KEY+'s')
We found it in production system and fixed in https://github.com/opsway/zohobooks-api/commit/b8d27a8f630cec888614c33ca0c19a0d2b355b4e